apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx

```diff
@@ -37,10 +40,9 @@ const AFE_CONSENT_BODY = (
 const CSTA_PRIVACY_POLICY_URL = 'https://csteachers.org/privacy-policy/';
 const CSTA_CONSENT_BODY = (
   <span>
-    I give Code.org permission to share my name and email address, and my
-    school's name, address, and NCES ID, with the Computer Science Teachers
-    Association (required if you want a CSTA+ membership). I provide my consent
-    to the use of my personal data as described in the{' '}
+    I opt-in for a free CSTA+ membership and access to Amazon webinars and
```

Comment:
Should this be translateable as well?

Reply:
Translating this form has been deprioritized, so I was selfishly translating as it was easy to. I think I didn't do this one because there was already existing code surrounding the privacy policy portion🫠

Reply:
Got it! If we're not prioritizing translation, I think it's good as is 👍